q2_diversity/_alpha/_visualizer.py

```diff
@@ -16,20 +16,16 @@
 import scipy
 import numpy as np
 import pandas as pd
-import qiime2
 from statsmodels.sandbox.stats.multicomp import multipletests
-import q2templates
 import biom
 import itertools
+
+import qiime2
+import q2templates
+from q2_diversity import _alpha
```

Comment:
This is one step away from creating a circular import nightmare, how about

```suggestion
from .pipeline import (...bleh)
```

Reply:
resolved in 960a7f0